src/RazorSdk/SourceGenerators/RazorSourceGenerationContext.cs

```diff
@@ -36,16 +36,6 @@ internal class RazorSourceGenerationContext
         /// </summary>
         public bool WaitForDebugger { get; private set; }
 
-        /// <summary>
```

Comment:
I think we can keep this if it helps debugging. It's not enabled in the IDE which is primarily what they were concerned about. If you'd like we can rename this to have a `TestOnly` prefix

Reply:
Sorry for the late commentary, but I think we should reconsider this.

I think it's important to have the option to generate the source code, specially when it's disabled by default. Otherwise the current experience degrades a lot:
* There are important integration aspects that we are not able to test.
* Ourselves and customers look up at the generated code in many cases when we can't troubleshoot an issue.

What's the motivation behind completely removing it, I think a much better place is to leave it optional as it was up to now.

Reply:
@javiercn We'll be bringing this back -- but using the official compiler support for this feature instead of rolling our own file writes. We're tracking this in https://github.com/dotnet/aspnetcore/issues/30302.

Reply:
Ok, thanks. Although it's not exactly the same since I believe that triggers generation for all the source generators, not just the blazor one.